src/repository/resources/asres.rs

```diff
@@ -405,6 +410,11 @@ impl AsBlocks {
 /// # Set operations
 ///
 impl AsBlocks {
+    /// Returns whether this AsBlocks contains an ASN.
```

Comment:
‘… a given ASN‘? – I kind of dislike having arguments referenced in the headline since it shows in the overview where you may or may not see a list of arguments.